subsys/bluetooth/host/att.c

```diff
@@ -188,6 +188,15 @@ static int chan_send(struct bt_att_chan *chan, struct net_buf *buf,
 			return -EAGAIN;
 		}
 
+		/* Check the encryption level for EATT */
+		if (IS_ENABLED(CONFIG_BT_EATT) &&
+			chan->att->conn->sec_level < BT_SECURITY_L2) {
```

Comment:
Minor: please fix the split line alignment: should be 0-n tabs + 0-7 spaces to align with the opening parenthesis of the first line (in this case 4 spaces, I believe)